ArduCopter/gps_glitch.pde

```diff
@@ -17,6 +17,14 @@ static void gps_glitch_update() {
     }
 }
 
+static void gps_glitch_mode_change_commanded(uint8_t mode_commanded)
+{
+    // ensure that commanded mode switches to ALT_HOLD will cancel the switch back to FLY
+    if (mode_commanded == ALT_HOLD) {
```

Comment:
Oh yeah, I forgot that shots change mode to Guided. The PR is fine, it was just a matter of having the same principle applied in the same place.